walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -24,9 +70,11 @@ img {
 
 .header-container {
   align-items: center;
+  background-color: grey;
   display: flex;
   height: 100px;
   justify-content: space-evenly;
+  margin-bottom: 10px;
   margin-right: auto;
   margin-left: auto;
```

Comment:
at this point you might want to combine to `margin: 0 auto 10px auto;` (not sure what you want for margin-top)

Reply:
done